### DISCUSSION TITLE
Discussion for community meeting (2022/11/23) | 社区会议讨论

### DISCUSSION THREAD

**Discussion by sczyh30** [alibaba/Sentinel#2968] [General]
Discussion for the biweekly meeting of the OpenSergo & Sentinel community (2022/11/23)

---

本次 Sentinel 社区会议将于 2022/11/23（周三）与 OpenSergo 社区双周会合并进行，欢迎大家预留时间参与，也欢迎大家发起 topic 来分享。议程如下：

* 同步上一周期的进展，包括微服务治理 spec 建设、Control Plane && SDK 进展、社区生态接入，以及 Sentinel 社区进展。
* 社区分享与讨论
  * Sentinel 2.0 新模型及流量治理骨架设计讨论 (Draft)
  * OpenSergo 流量路由 spec 订正讨论：基于 Istio VirtualService/DestinationRule 进行扩展
  * OpenSergo 与 Istio 生态结合的方案讨论

主题：OpenSergo & Sentinel 社区双周会（钉钉群会议）
时间：2022年11月23日 周三 19:30 - 20:30
入会口令：244 801 51010
电话呼入：+862281944261（中国大陆）+867388953916（中国大陆）
入会链接：dingtalk://dingtalkclient/page/videoConfFromCalendar?confId=65d3c04b-b69e-44ce-b310-35054cff1d57&calendarId=2053613768

Discussion for OpenSergo: https://github.com/opensergo/opensergo-specification/discussions/59

**Comment by sczyh30** [comment]
Summary refer: https://github.com/opensergo/opensergo-specification/discussions/59#discussioncomment-4233146

**Comment by sczyh30** [comment]
Sentinel 社区关键进展:

* 近期社区开发者贡献了 Kubernetes operator for Sentinel dashboard，欢迎大家一起 review: https://github.com/sentinel-group/sentinel-dashboard-k8s-operator/pull/1
* 社区正在梳理并处理 Sentinel 2.0 milestone issues，欢迎社区添加感兴趣的 feature/enhancement：https://github.com/alibaba/Sentinel/milestone/18

**Comment by losikurya** [comment]
Greatings!
